fenix/app/src/androidTest/java/org/mozilla/fenix/onboarding/view/JunoOnboardingMapperTest.kt

```diff
@@ -88,7 +293,49 @@ private val defaultBrowserCardData = OnboardingCardData(
     primaryButtonLabel = StringHolder(null, "default browser primary button text"),
     secondaryButtonLabel = StringHolder(null, "default browser secondary button text"),
     ordering = 10,
+    prerequisites = listOf("ALWAYS"),
+    disqualifiers = listOf("NEVER"),
 )
+
+private val defaultBrowserCardDataNoConditions = OnboardingCardData(
+    cardType = OnboardingCardType.DEFAULT_BROWSER,
+    imageRes = R.drawable.ic_onboarding_welcome,
+    title = StringHolder(null, "default browser title"),
+    body = StringHolder(null, "default browser body with link text"),
+    linkText = StringHolder(null, "link text"),
+    primaryButtonLabel = StringHolder(null, "default browser primary button text"),
+    secondaryButtonLabel = StringHolder(null, "default browser secondary button text"),
+    ordering = 10,
+    prerequisites = listOf("ALWAYS"),
+    disqualifiers = listOf("NEVER"),
+)
```

Comment:
Think this can be removed as it is the same as `defaultBrowserCardData` and  `addSearchWidgetCardDataNoConditions` has this scenario covered?